src/test/java/seedu/address/logic/commands/FindProfileCommandTest.java

```diff
@@ -34,41 +37,61 @@ public void equals() {
         NameContainsKeywordsPredicate secondPredicate =
                 new NameContainsKeywordsPredicate(Collections.singletonList("second"));
 
-        FindCommand findFirstCommand = new FindCommand(firstPredicate);
-        FindCommand findSecondCommand = new FindCommand(secondPredicate);
+        FindProfileCommand findProfileFirstCommand = new FindProfileCommand(firstPredicate);
+        FindProfileCommand findProfileSecondCommand = new FindProfileCommand(secondPredicate);
 
         // same object -> returns true
-        assertTrue(findFirstCommand.equals(findFirstCommand));
+        assertTrue(findProfileFirstCommand.equals(findProfileFirstCommand));
 
         // same values -> returns true
-        FindCommand findFirstCommandCopy = new FindCommand(firstPredicate);
-        assertTrue(findFirstCommand.equals(findFirstCommandCopy));
+        FindProfileCommand findProfileFirstCommandCopy = new FindProfileCommand(firstPredicate);
+        assertTrue(findProfileFirstCommand.equals(findProfileFirstCommandCopy));
 
         // different types -> returns false
-        assertFalse(findFirstCommand.equals(1));
+        assertFalse(findProfileFirstCommand.equals(1));
 
         // null -> returns false
-        assertFalse(findFirstCommand.equals(null));
+        assertFalse(findProfileFirstCommand.equals(null));
 
         // different profile -> returns false
-        assertFalse(findFirstCommand.equals(findSecondCommand));
+        assertFalse(findProfileFirstCommand.equals(findProfileSecondCommand));
     }
 
     @Test
     public void execute_zeroKeywords_noProfileFound() {
-        String expectedMessage = String.format(MESSAGE_PROFILES_LISTED_OVERVIEW, 0);
+        String expectedMessage = String.format(MESSAGE_NO_MATCH);
         NameContainsKeywordsPredicate predicate = preparePredicate(" ");
-        FindCommand command = new FindCommand(predicate);
+        FindProfileCommand command = new FindProfileCommand(predicate);
         expectedModel.updateFilteredProfileList(predicate);
         assertCommandSuccess(command, model, expectedMessage, expectedModel);
         assertEquals(Collections.emptyList(), model.getFilteredProfileList());
     }
 
+    @Test
+    public void execute_singleKeyword_singleProfileFound() {
+        String expectedMessage = String.format(MESSAGE_PROFILE_LISTED_OVERVIEW, 1);
```

Comment:
No need for `String.format` here, the `MESSAGE_PROFILE_LISTED_OVERVIEW` doesn't take in any parameters.